foreman/Dockerfile.tests

```diff
@@ -0,0 +1,21 @@
+# This Dockerfile is meant to be run from the directory above where it resides.
```

Comment:
That `WORKDIR` is referring to a directory within the docker container, not a location on the host machine. See https://docs.docker.com/engine/reference/builder/#workdir for more information.